Library/Homebrew/formula.rb

```diff
@@ -1624,6 +1627,7 @@ def to_hash
       "optional_dependencies"    => dependencies.select(&:optional?)
                                                 .map(&:name)
                                                 .uniq,
+      "macos_dependencies"       => macos_deps.uniq,
```

Comment:
Would be good to use a name more like `uses_from_macos` that points more to the DSL.

Reply:
✅